CHANGELOG.md

```diff
@@ -120,7 +122,7 @@
 * XCAssets: renamed the catalogs we use for sample code & testing to avoid some confusion.  
   [David Jennes](https://github.com/djbe)
   [#613](https://github.com/SwiftGen/SwiftGen/pull/613)
-* Update to SwiftLint 0.31.0 and enable some extra SwiftLint rules.  
+* Update SwiftLint and enable some extra SwiftLint rules.  
```

Comment:
Not sure when but we later updated to SwiftLint 0.39.2, so I figured we shouldn't let people know when reading the CHANGELOG that we're still on 0.31